Classes/Core/Controllers/FLEXNavigationController.m

```diff
@@ -148,6 +148,13 @@ - (void)handleNavigationBarSwipe:(UISwipeGestureRecognizer *)sender {
 }
      
 - (void)handleNavigationBarTap:(UIGestureRecognizer *)sender {
+    CGPoint location = [sender locationInView:self.navigationBar];
+    UIView *hitView = [self.navigationBar hitTest:location withEvent:nil];
+
+    if ([hitView isKindOfClass:[UIControl class]]) {
+        return;
+    }
```

Comment:
```suggestion
    // Don't reveal the toolbar if we were just tapping a button
    CGPoint location = [sender locationInView:self.navigationBar];
    UIView *hitView = [self.navigationBar hitTest:location withEvent:nil];
    if ([hitView isKindOfClass:[UIControl class]]) {
        return;
    }
```

The diff looks really weird but I just wanted to remove the empty line and add a comment